src/SignalR/server/Core/src/HubOptions.cs

```diff
@@ -51,5 +61,8 @@ public class HubOptions
         /// Gets or sets the max buffer size for client upload streams. The default size is 10.
         /// </summary>
         public int? StreamBufferCapacity { get; set; } = null;
+
+        internal long? _maximumReceiveMessageSize { get; set; } = null;
```

Comment:
Nit: Don't we reserver the `_` prefix for private fields? Weird to see an internal property.